specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/examples/PostDeployWorkflowArtifacts.json

```diff
@@ -0,0 +1,51 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testrg123",
+    "name": "testsite2",
+    "workflowName": "stateful2",
+    "slot": "staging",
+    "api-version": "2021-01-01"
+  },
+  "request": {
+    "body": {
+      "files": {
+        "stateful1/workflow.json": {
+          "definition": {
+            "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
+            "actions": {},
+            "contentVersion": "1.0.0.0",
+            "outputs": {},
+            "parameters": {},
+            "triggers": {}
+          }
+        },
+        "connections.json": {
+          "managedApiConnections": {
+            "office365": {
+              "api": {
+                "id": "string"
+              },
+              "connection": {
+                "id": "string"
+              },
+              "connectionRuntimeUrl": "string",
+              "authentication": {
+                "type": "Raw",
+                "scheme": "Key",
+                "parameter": "@appsetting('office365-connectionKey')"
```

Comment:
>ter": "@appsetting('office365-connectionKey')" [](http://example.com/codeflow?start=23&length=46)

Would this be a valid example since this key doesn't look declared in the appsettings in this request? Valid examples are better examples!

Reply:
Updated to be a valid example :)